diff --git a/node_modules/gatsby-theme-ghost-members/src/components/common/SubscribeForm.js b/node_modules/gatsby-theme-ghost-members/src/components/common/SubscribeForm.js
index ecfee44..598e279 100644
--- a/node_modules/gatsby-theme-ghost-members/src/components/common/SubscribeForm.js
+++ b/node_modules/gatsby-theme-ghost-members/src/components/common/SubscribeForm.js
@@ -32,7 +32,7 @@ class SubscribeForm extends React.Component {
         event.preventDefault()
 
         const cmsUrl = this.props.url
-        const postURL = `${cmsUrl}/members/api/send-magic-link/`
+        const postURL = `${cmsUrl}/ghost/api/canary/members/send-magic-link`
 
         const values = {
             email: this.state.value,
diff --git a/node_modules/gatsby-theme-ghost-members/src/gatsby-theme-try-ghost/components/common/effects/OverlayContainer.js b/node_modules/gatsby-theme-ghost-members/src/gatsby-theme-try-ghost/components/common/effects/OverlayContainer.js
index 4a04cf8..1f32561 100644
--- a/node_modules/gatsby-theme-ghost-members/src/gatsby-theme-try-ghost/components/common/effects/OverlayContainer.js
+++ b/node_modules/gatsby-theme-ghost-members/src/gatsby-theme-try-ghost/components/common/effects/OverlayContainer.js
@@ -40,7 +40,7 @@ class OverlayContainer extends React.Component {
         const settings = this.props.data.allGhostSettings.edges[0].node
 
         const cmsUrl = settings.url
-        const postURL = `${cmsUrl}/members/api/send-magic-link/`
+        const postURL = `${cmsUrl}/ghost/api/canary/members/send-magic-link`
 
         const values = {
             email: this.state.value,
